src/w2grid.js

```diff
@@ -704,6 +685,66 @@
             }
             return time;
         },
+    
+        compareRecords: function (a, b) {
```

Comment:
I think it would be better to avoid creating methods on the grid levels for internal use (if possibly). For example this method is called compare 2 records for sorting purposes and it returns info expected by array.sort and uses internal vairables of the grid such as grid.sortData. It is unlikely that it would ever make sense for the developer who just consumes grid to call it. Therefore, it should be within localSort function and not accessible for the developer.